js/everything.js

```diff
@@ -357,6 +364,16 @@ function getCurrentVideoID() {
     return v;
 }
 
+function getCurrentTimePosition() {
+    var t = parseInt(getParameterByName(window.location.search, "t"), 10);
+    if (t >= 0 && t < 999999) {
+        return t;
+    }
+    else {
```

Comment:
We don't even need this `else` since the return value will be `undefined` anyways